src/broadcast_receiver.cpp

```diff
@@ -32,14 +32,10 @@ void BroadcastReceiver::proceedReceivedData(const uint8_t* buffer,
     {
         case TESTER_PRESENT_REQ:
         {
-            if(buffer[1] == 0x80 && num_bytes >= 2)
-            {
-                pUdsReceiver_->pSessionCtrl_->reset();
-            }
-            else
+            if(buffer[1] != 0x80)
```

Comment:
I would leave the num_bytes >= 2 but rather put it before the check of the buffer to avoid segfault if someone sends a TesterPresent without second byte.